### PR TITLE
Update README.md

### LINKED ISSUE #10404: Errors in readme
spelling/grammar errors in readme

### PR DESCRIPTION
**Pull request by ghost**
spelling/grammar
Fixes #10404

### CODE REVIEW

**Comment by sprintr** [comment]
You have to sign [Brackets CLA](http://dev.brackets.io/brackets-contributor-license-agreement.html) first.

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ How to install and run Brackets
 
 Installers for the latest stable build for Mac, Windows and Linux (Debian/Ubuntu) can be [downloaded here](http://brackets.io/).
 
-The Linux version has most of the features of the Mac and Windows versions, but
+The Linux version has more features than the Mac and Windows versions, but
```
Comment (redmunds) [inline README.md]: This sentence is trying to say "Most, but not all, of the features implemented for Mac and Windows versions have been implemented for Linux". It is *not* true the "Linux has more features".

Comment (Mark-Simulacrum) [inline README.md]: This change changes the meaning of the sentence: it used to be that only some of the features of the Mac/Windows versions are present in the Linux version. Now, more features are present in the Linux version than the Mac/Windows version.

Is this the case? I would doubt that - I would suspect that Mac and Windows have more features than the Linux version.

Comment (peterflynn) [inline README.md]: Maybe a better change would be: "The Linux version has almost all the features found in the Mac and Windows versions, but..."

**Comment by peterflynn** [comment]
Closing since this PR changes the meaning of the sentence in a way that seems incorrect.

**Pull request closed without merging**